### ISSUE TITLE
Cannot pull java:openjdk-8u66-jdk because of aufs error

### ISSUE THREAD

**Issue by dcoraboeuf** [docker-library/java#56]
Hi,

Using version 1.9.0 of Docker:

```
> docker version
Client:
 Version:      1.9.0
 API version:  1.21
 Go version:   go1.4.3
 Git commit:   76d6bc9
 Built:        Tue Nov  3 19:20:09 UTC 2015
 OS/Arch:      darwin/amd64

Server:
 Version:      1.9.0
 API version:  1.21
 Go version:   go1.4.3
 Git commit:   76d6bc9
 Built:        Tue Nov  3 19:20:09 UTC 2015
 OS/Arch:      linux/amd64
```

If I try to pull the JDK8u66:

```bash
docker pull java:openjdk-8u66-jdk
```

I get the following error:

```
de4a13c84f53: Error pulling image (openjdk-8u66-jdk) from docker.io/library/java, Driver aufs failed to create image rootfs dc921aeac8d06e7bd5b2d85b8684bb7c94ed08db3f18a9b6e897827810b289de: open /mnt/sda1/var/lib/docker/aufs/layers/4c0cc976b7bbefe9abdfd8f0aae1bb3759004d3c9888c3164b38909b862c0cdf: no such file or directory 4b38909b862c0cdf: no such file or directory 
371b8d572221: Download complete 
dc921aeac8d0: Error downloading dependent layers 
Error pulling image (openjdk-8u66-jdk) from docker.io/library/java, Driver aufs failed to create image rootfs dc921aeac8d06e7bd5b2d85b8684bb7c94ed08db3f18a9b6e897827810b289de: open /mnt/sda1/var/lib/docker/aufs/layers/4c0cc976b7bbefe9abdfd8f0aae1bb3759004d3c9888c3164b38909b862c0cdf: no such file or directory
```

I can reproduce this error on MacOS Docker Toolbox or in an Ubuntu system.

Downloading the JDK7 does not raise any issue:

```bash
> docker pull java:openjdk-7u85-jdk
openjdk-7u85-jdk: Pulling from library/java
a719479f5894: Pull complete 
91bac885982d: Pull complete 
8ca313dfa777: Pull complete 
66769fc86d84: Pull complete 
3b907d566478: Pull complete 
ae047c5d2657: Pull complete 
4b8cc38cff38: Pull complete 
0a1a4eb4aa08: Pull complete 
76382213662a: Pull complete 
Digest: sha256:d0b821d0cbeb06508c41cb82a06311c1ea177512b334896db944e3b74602574a
Status: Downloaded newer image for java:openjdk-7u85-jdk
```

No problem either for 8u45:

```bash
> docker pull java:openjdk-8u45-jdk
openjdk-8u45-jdk: Pulling from library/java
3d7d0c417e1d: Pull complete 
8b5de14bc579: Pull complete 
003124e47cbd: Pull complete 
e167c026a01b: Pull complete 
229aba7c8c65: Pull complete 
a96cd57ee59b: Pull complete 
a255cfd41224: Pull complete 
957224005b89: Pull complete 
e892b41b6262: Pull complete 
93944342944a: Pull complete 
f51a5a26e10a: Pull complete 
26150110f4d8: Pull complete 
Digest: sha256:df6baeaa694ae52f9df00115bffc5f8bbd53b2c724c13ccbeb1951aea48cb626
Status: Downloaded newer image for java:openjdk-8u45-jdk
```

Any hint of the source of the issue?

Thanks,
Damien.

**Comment by tianon** [comment]
This looks very similar to https://github.com/docker-library/python/issues/66, and unlike that one I can still reproduce this one. :cry:

**Comment by dcoraboeuf** [comment]
Very similar to the problem mentioned in docker/docker#18008 Hoping for a 1.9.1 which would fix the issue.

**Comment by dcoraboeuf** [comment]
This seems now to be fixed, without me having updated any client environment...

**Comment by tianon** [comment]
That's great news!  The issue was something about how the manifest was
pushed, so I think we fixed it by repushing all images with 1.8.3. :+1:

**Comment by dcoraboeuf** [comment]
Excellent. I'm closing the issue.
